frontend/cypress/support/helper/dom-helper/dialogs/objectiveDialog.ts

```diff
@@ -32,6 +32,13 @@ export default class ObjectiveDialog extends Dialog {
     cy.getByTestId('save-draft').click();
   }
 
+  excludeKeyResults(keyResults: string[]) {
+    keyResults.forEach((keyResult) => {
+      cy.get('label').contains(keyResult.slice(0, 30)).click();
```

Comment:
The cypress viewport is not able to show the whole name of the KeyResults because they get truncated once they reach the width of one line. Since some of these KeyResults have a pretty long name I did not want to take any risk.